configure.ac

```diff
@@ -80,26 +80,46 @@ AC_ARG_WITH(gtk,
     [ac_with_gtk="${withval}"],
     [ac_with_gtk=2])
 
-# Checks for libraries.
-
-PKG_CHECK_MODULES(XLIB, "x11")
-AC_SUBST(XLIB_CFLAGS)
-AC_SUBST(XLIB_LIBS)
+AC_ARG_ENABLE(wayland,
+    [AC_HELP_STRING([--enable-wayland],
+        [Enable Wayland support])],
+    [enable_wayland="${enableval}"],
+    [enable_wayland=no])
+
+AC_ARG_ENABLE(x11,
+    [AC_HELP_STRING([--enable-x11],
+        [Enable X11 support])],
+    [enable_x11="${enableval}"],
+    [enable_x11=yes])
+
+if test "x$enable_wayland" = "xno" && test "x&$enable_x11" = "xno"; then
```

Comment:
`if test "x$enable_wayland" = "xno" -a "x&$enable_x11" = "xno"; then`
is shorter